pkg/manifests/rte/rte.go

```diff
@@ -183,6 +192,8 @@ func (mf Manifests) ToCreatableObjects(hp *deployer.Helper, log tlog.Logger) []d
 	return append(objs,
 		deployer.WaitableObject{Obj: mf.Role},
 		deployer.WaitableObject{Obj: mf.RoleBinding},
+		deployer.WaitableObject{Obj: mf.ClusterRole},
+		deployer.WaitableObject{Obj: mf.ClusterRoleBinding},
```

Comment:
when we transition to RTE >= v0.3.0 we will need to get rid of the role and rolebinding
We are not there yet

Reply:
Why do we need to get rid of the role and role binding, otherwise I think we grant broad permissions to get pod and update pod status under the RTE manifest https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/e57792df4a3b19928dbedf9cb23684f05acc1397/manifests/resource-topology-exporter-rbac.yaml#L15.
We should only have permissions to update daemon set pods under the specific namespace we do not need cluster scope role for it.

Reply:
right, good point. I guess we will need them both after all.